salt/states/linux_acl.py

```diff
@@ -128,6 +130,11 @@ def absent(name, acl_type, acl_name='', perms='', recurse=False):
            'changes': {},
            'comment': ''}
 
+    if not os.path.isfile(name):
```

Comment:
Please note that ACLs can be set on any kind of file system objects, not just files. In the issue I've filed that started this (#35591) the entry in question was in fact a directory.

Therefore this test is overly restrictive.